pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/model/Style.java

```diff
@@ -0,0 +1,26 @@
+package net.sourceforge.pmd.util.fxdesigner.model;
+
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
+
+public class Style {
+
+
+    public static TextFlow highlight(String text, String filter ) {
```

Comment:
This highlight method won't scale to using a different algorithm for matching inputs. It only works when the algorithm is a simple contains check, and when the only highlighted span is the whole filter. Using a static class also isn't a good idea